### DISCUSSION TITLE
mirai 各个模块之前的关系

### DISCUSSION THREAD

**Discussion by huiqiangyang** [mamoe/mirai#881] [Q&A] [answered]
### mirai 有很多个模块，各个模块之前的关系是什么样的，mirai-core， mirai-core-jvm，mirai-core-qqandroid-jvm，mirai-core-android 有什么差异？

![image](https://user-images.githubusercontent.com/13185035/105277198-eb638500-5bdd-11eb-8bce-6901ecce0e0f.png)

**Comment by Him188** [comment, marked as answer]
Kotlin MPP，关系很复杂，一般用户不需要知道细节。只需要知道 mirai [文档里描述的那些内容](https://github.com/mamoe/mirai/blob/dev/docs/ConfiguringProjects.md)就行

**Reply by huiqiangyang** [reply]
好吧，只是喊 牛逼就对了
